index.html

```diff
@@ -121,12 +121,10 @@ <h3 id="filterHeader">Filter</h3>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <!-- Compiled and minified JavaScript -->
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
-    <script src="j.js"></script>
```

Comment:
You can delete this line, I renamed the JS file to mmlapis.js.
(We don't need the j.js file anymore.)